sway/ipc-server.c

```diff
@@ -276,6 +277,7 @@ void ipc_get_pixels(wlc_handle output) {
 		ipc_send_reply(req->client, data, len);
 		free(data);
 		// free the request since it has been handled
+		ipc_client_disconnect(req->client);
```

Comment:
It's not necessary to disconnect the client here. They can stay connected and issue further commands.